### DISCUSSION TITLE
Create New Asset Folders at least 3 characters

### DISCUSSION THREAD

**Discussion by djlongy** [requarks/wiki#4038] [Error / Bug Report] [answered]
I can't create a new asset folder named ui, replicating your example such as /assets/**ui**/ui-assets-dialog.png
This should be possible seeing as the documentation states:
Must be at least 2 characters in length.

It has to be three characters in order to let me create the folder.
Please reduce this to 2 characters in length.

Tested using the latest version 2.5.201

**Comment by NGPixel** [comment, marked as answer]
My example has an `assets` folder as the root folder, not `ui`. The restriction only applies to the root folder.

**Reply by djlongy** [reply]
It's not working even as a subfolder
